spec/api/management.yaml

```
@@ -14,29 +14,30 @@ components:
     # Deletion is normally automatic (not through API) when the trust bundle contents
     # indicate that it has expired
 
-    Organization:
-      # An organization is the building block to create Federation Groups. It provides multitenancy capabilities.
-      type: object
-      properties:
-        id:
-          type: integer
-          format: int64
-        name:
-          type: string
-          format: string
-      required:
-        - id
-        - name
+#    Organization:
+#      # An organization is the building block to create Federation Groups. It provides multitenancy capabilities.
+#      type: object
+#      properties:
+#        id:
+#          type: integer
+#          format: int64
+#        name:
+#          type: string
+#          format: string
+#      required:
+#        - id
+#        - name
     FederationGroup:
-      # A Federation Group is a group of SpireServers (representation of the bridge)
+      # A Federation Group represents a bridge for its Members,
+      # allowing relationships to be established
       type: object
       properties:
         id:
           type: integer
           format: int64
```

```suggestion
          type: string
          format: uuid
```
@jufantozzi maybe we could use format: uuid to store the ID?

---

how does uuid's translates to the different dbms?

---

I can say that [gorm](https://medium.com/@the.hasham.ali/how-to-use-uuid-key-type-with-gorm-cc00d4ec7100) supports it. [MongoDB](https://www.mongodb.com/docs/manual/reference/method/UUID/) seems too